### PR TITLE
Changed default version values when launching hand in simulation

### PR DESCRIPTION
**Pull request by maxzieba**
## Proposed changes

User Story: https://shadowrobot.atlassian.net/browse/SRC-7025
Changed default version values when launching hand in simulation.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [X] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [ ] Manually tested on hardware (if hardware specific or related).

## Documentation

- [X] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: sr_robot_launch/launch/sr_left_ur10arm_hand.launch [hunk 1]
```diff
@@ -59,11 +59,14 @@
...
   <arg name="side" value="left"/>
-
   <arg name="hand_type" default="hand_e"/>
-  <arg name="hand_version" default="E3M5"/>
-  <arg name="fingers" default="th,ff,mf,rf,lf"/>
-  <arg name="tip_sensors" default="th=pst,ff=pst,mf=pst,rf=pst,lf=pst"/>
+  <arg if="$(eval arg('hand_type') == 'hand_e')" name="hand_version" default="E3M5"/>
+  <arg if="$(eval arg('hand_type') == 'hand_g')" name="hand_version" default="G1M5"/>
+  <arg if="$(eval arg('hand_type') == 'hand_c')" name="hand_version" default="C6M2"/>
+  <!-- Use all available - allowed values are a list of th,ff,mf,rf,lf -->
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_left_ur10arm_hand.launch]:
```suggestion
  <!-- Set to all allowed fingers by default but you can specify less using a list like this "th,ff,mf,rf,lf" -->
```

File: sr_robot_launch/launch/sr_left_ur10arm_hand.launch [hunk 2]
```diff
@@ -59,11 +59,14 @@
...
+  <arg name="fingers" default="all"/>
+  <!-- Use pst on all - allowed values are a list of th=pst,ff=pst,mf=pst,rf=pst,lf=pst -->
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_left_ur10arm_hand.launch]:
```suggestion
  <!-- Set to pst sensors by default but they can be specified by finger using a list like this "th=pst,ff=pst,mf=pst,rf=pst,lf=pst" -->
```

File: sr_robot_launch/launch/sr_ur_arm_hand.launch [hunk 3]
```diff
@@ -68,13 +68,17 @@
 
   <!-- SIMULATED ROBOTS -->
   <arg name="side" default="right"/>
-  <arg name="hand_version" default="E3M5"/>
-  <arg name="fingers" default="th,ff,mf,rf,lf"/>
-  <arg name="tip_sensors" default="th=pst,ff=pst,mf=pst,rf=pst,lf=pst"/>
+  <arg name="hand_type" default="hand_e"/>
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_ur_arm_hand.launch:71]: This is a launch file that will be called by the others. I am wondering if we really need the ifs here. maybe set the defaulsts to hand e with all and pst?

File: sr_robot_launch/launch/sr_ur_arm_hand.launch [hunk 4]
```diff
@@ -68,13 +68,17 @@
...
+  <arg if="$(eval arg('hand_type') == 'hand_e')" name="hand_version" default="E3M5"/>
+  <arg if="$(eval arg('hand_type') == 'hand_g')" name="hand_version" default="G1M5"/>
+  <arg if="$(eval arg('hand_type') == 'hand_c')" name="hand_version" default="C6M2"/>
+  <!-- Use all available - allowed values are a list of th,ff,mf,rf,lf -->
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_ur_arm_hand.launch]: replace with first suggestion

File: sr_robot_launch/launch/sr_ur_arm_hand.launch [hunk 5]
```diff
@@ -68,13 +68,17 @@
...
+  <arg name="fingers" default="all"/>
+  <!-- Use pst on all - allowed values are a list of th=pst,ff=pst,mf=pst,rf=pst,lf=pst -->
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_ur_arm_hand.launch]: replace with first suggestion

File: sr_robot_launch/launch/srhand.launch [hunk 6]
```diff
@@ -41,9 +41,13 @@
   <arg if="$(eval arg('hand_id') == 'lh')" name="side" default="left"/>
   <!-- Set to hand_e or hand_g -->
   <arg name="hand_type" default="hand_e"/>
-  <arg name="hand_version" default="E3M5"/>
-  <arg name="fingers" default="th,ff,mf,rf,lf"/>
-  <arg name="tip_sensors" default="th=pst,ff=pst,mf=pst,rf=pst,lf=pst"/>
+  <arg if="$(eval arg('hand_type') == 'hand_e')" name="hand_version" default="E3M5"/>
+  <arg if="$(eval arg('hand_type') == 'hand_g')" name="hand_version" default="G1M5"/>
+  <arg if="$(eval arg('hand_type') == 'hand_c')" name="hand_version" default="C6M2"/>
+  <!-- Use all available - allowed values are a list of th,ff,mf,rf,lf -->
```
Comment (beatrizleon) [inline sr_robot_launch/launch/srhand.launch]: replace with first suggestion

File: sr_robot_launch/launch/srhand.launch [hunk 7]
```diff
@@ -41,9 +41,13 @@
...
+  <arg name="fingers" default="all"/>
+  <!-- Use pst on all - allowed values are a list of th=pst,ff=pst,mf=pst,rf=pst,lf=pst -->
```
Comment (beatrizleon) [inline sr_robot_launch/launch/srhand.launch]: replace with first suggestion

File: sr_robot_launch/launch/sr_right_ur10arm_hand.launch [hunk 8]
```diff
@@ -59,11 +59,14 @@
...
   <arg name="side" value="right"/>
-
   <arg name="hand_type" default="hand_e"/>
-  <arg name="hand_version" default="E3M5"/>
-  <arg name="fingers" default="th,ff,mf,rf,lf"/>
-  <arg name="tip_sensors" default="th=pst,ff=pst,mf=pst,rf=pst,lf=pst"/>
+  <arg if="$(eval arg('hand_type') == 'hand_e')" name="hand_version" default="E3M5"/>
+  <arg if="$(eval arg('hand_type') == 'hand_g')" name="hand_version" default="G1M5"/>
+  <arg if="$(eval arg('hand_type') == 'hand_c')" name="hand_version" default="C6M2"/>
+  <!-- Use all available - allowed values are a list of th,ff,mf,rf,lf -->
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_right_ur10arm_hand.launch]: replace with first suggestion

File: sr_robot_launch/launch/sr_right_ur10arm_hand.launch [hunk 9]
```diff
@@ -59,11 +59,14 @@
...
+  <arg name="fingers" default="all"/>
+  <!-- Use pst on all - allowed values are a list of th=pst,ff=pst,mf=pst,rf=pst,lf=pst -->
```
Comment (beatrizleon) [inline sr_robot_launch/launch/sr_right_ur10arm_hand.launch]: replace with first suggestion

**Review by beatrizleon — CHANGES_REQUESTED** [review]

File: sr_robot_launch/launch/sr_ur_arm_hand.launch (hunk 3 above)
Comment (maxzieba) [reply sr_robot_launch/launch/sr_ur_arm_hand.launch:71]: I am setting only the defaults, if the arguments are set from a top-level launch file, the values within this one will be overridden. With defaults and the ifs, it's possible to launch only this as a standalone without the issue of type&version mismatch.

**Review by beatrizleon — APPROVED** [review]

**Review by biofotis — APPROVED** [review]

**Pull request merged**